WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -7,6 +7,8 @@ import Kanvas
 
 class GutenbergViewController: UIViewController, PostEditor, FeaturedImageDelegate {
 
+
+
```

Comment:
We can remove the extra lines here

Reply:
Resolved w/ https://github.com/wordpress-mobile/WordPress-iOS/pull/17429/commits/1bd6ff44927f35a724ec2013374a851b3a4f7386